e2e/whisper/whisper_mailbox_test.go

```diff
@@ -231,10 +230,395 @@ func (s *WhisperMailboxSuite) TestRequestMessageFromMailboxAsync() {
 
 }
 
-func (s *WhisperMailboxSuite) startBackend() (*api.StatusBackend, func()) {
-	//Start sender node
+func (s *WhisperMailboxSuite) TestRequestMessagesFromMailboxFromGroupChat() {
+	//Start mailbox, alice, bob, charlie node
+	mailboxBackend, stop := s.startMailboxBackend()
+	defer stop()
+	mailboxNode, err := mailboxBackend.NodeManager().Node()
+	s.Require().NoError(err)
+	mailboxEnode := mailboxNode.Server().NodeInfo().Enode
+
+	aliceBackend, stop := s.startBackend("alice")
```

Comment:
Lots of paragraphs have comment above their block, can we do something similar for these as well? ~L240-270